protocol/src/test/java/org/fidoalliance/fdo/protocol/ManufacturingInfoTest.java

```diff
@@ -0,0 +1,216 @@
+// package org.fidoalliance.fdo.protocol;
+
+// public class ManufacturingInfoTest {
+
+// }
+
+package org.fidoalliance.fdo.protocol;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.util.UUID;
+import org.apache.commons.codec.DecoderException;
+import org.fidoalliance.fdo.protocol.message.Guid;
+import org.fidoalliance.fdo.protocol.serialization.ManufacturingInfoDeserializer;
+import org.junit.jupiter.api.Test;
+
+public class ManufacturingInfoTest {
+    @Test
+    public void Test() throws DecoderException, IOException {
+
+        ManufacturingInfoDeserializer obj = new ManufacturingInfoDeserializer();
+        String[] invalidStrings = { "", "null", "none", "true", "false", "undefined", "NaN", "nil" };
+
+        String[] moreInvalidStrings = {
+                "",
+                "undefined",
+                "undef",
+                "null",
+                "NULL",
+                "(null)",
+                "nil",
+                "NIL",
+                "true",
+                "false",
+                "True",
+                "False",
+                "TRUE",
+                "FALSE",
+                "None",
+                "����8������?",
+                "�?�?i�¯?�",
+                "����?��?���?",
+                "����?����?��",
+                "`/���??������",
+                "????",
+                "???????????????????????????????????????????????????????????????????????????????",
+                "??????????",
+                "�45",
+                "�45012",
+                "??? ??? ???",
+                "'",
+                "\"",
+                "''",
+                "\"\"",
+                "'\"'",
+                "\"''''\"'\"",
+                "\"'\"'\"''''\"",
+                "<foo val=�bar� />",
+                "<foo val=�bar� />",
```

Comment:
Can you remove the redundant test cases ?

Reply:
https://github.com/fido-device-onboard/pri-fidoiot/pull/643/commits/193ccdca63c131aa9169d00ccb3d452e6d494ea1